### DISCUSSION TITLE
Hello, I made somethings, again ^^

### DISCUSSION THREAD

**Discussion by SMK-SonHR** [exult/exult#518] [Show and tell]
First of all, Thanx a lot 
especially, "thecommisar", "knightcaptainu7" 
you make me feel better. Because I think my works are too personal to share, so even u7 gamers would have no interest in my works.

so, I have somethings to show. but I didn't post on my blog yet, so no share now.

 
![2023-07-11_14_55_49](https://github.com/exult/exult/assets/152712037/e32a394c-a90e-4c60-9dda-8c01bda45ccb)

I made black horse cart :)

![2024-05-06 00 20 05](https://github.com/exult/exult/assets/152712037/8556ff6d-7292-4d94-abef-c08805898148)

my party. 

evil ryu
tyris (in golden axe game)
Sarevok (in Baldur's gate 1,2)

I hope these works would give some good influence to your mod :)
Thank you very much!!!

**Comment by thecommissar** [comment]
These are cool as hell. Love how good that Ryu shape is too. 

Love the horse cart, some good work there.

**Comment by SMK-SonHR** [comment]
maybe you give your e-mail, the, we can DM each other easily. Do I give my instagram? 
I don't know how can I help. 

well, I didn't have experience like this, so what can I do 4 u?
I am not good mod maker, I just made some shape for fun, for me, So, I'm afraid of that you will disappoint by my works.
so, my answer is "i don't know, my works can help you?"
If you want me, then I help you. But my reply could be slow. Please understand me. I'm father of two children. And woker. ^^:


and Now, I give you my works.(zip files)  It's free.


1. Barge set
 you saw my barge. 

2. Sarevok 
 I'm fan of Baldus's gate, I try make sarevok U7 version. is that good? kkk I don't know ^^:;;;;;

3. Diablo 
 I just try make diablo U7 version. just make color dark and swap dragon's head. (origin body is ether monster) 






yeah that's all ^^





-----Original Message-----

**Reply by SMK-SonHR** [reply]
[1. Barge.zip](https://github.com/exult/exult/files/15288930/1.Barge.zip)
[2. Sarevok_ver2.zip](https://github.com/exult/exult/files/15288932/2.Sarevok_ver2.zip)
[3. Diablo.zip](https://github.com/exult/exult/files/15288933/3.Diablo.zip)
![0017](https://github.com/exult/exult/assets/152712037/9d6b78fb-36ee-47ef-8423-6a4886a21ece)

**Comment by thecommissar** [comment]
Oh yeah, my IG is hhkanino, if you send me a message there we can chat about my ideas
________________________________
